Project/Assets/Editor/CustomInspectors/TwineNodeEditor.cs

```diff
@@ -7,16 +7,24 @@
 [CustomEditor(typeof(TwineNode))]
 public class TwineNodeEditor : Editor {
 
-
 	TwineNode node;
+    GameObject[] editorTriggeredObjects = new GameObject[0];
 
-	public override void OnInspectorGUI ()
+    public override void OnInspectorGUI ()
 	{
 		node = (TwineNode)target;
 
-		node.isDecisionNode = EditorGUILayout.Toggle ("Decision node?", node.isDecisionNode);
-		node.objectsToTrigger = PrairieGUI.drawObjectList ("Objects To Trigger", node.objectsToTrigger);
-		EditorGUILayout.LabelField ("Name", node.name);
+        if (editorTriggeredObjects.Length == 0)
```

Comment:
Is this the best condition to check for a sync the editor array to the target's array? What if the actual length of the list was 0? Then this will be constantly evaluating to `true` and reassigning the `editorTriggeredObjects` array.

It seems to me this is only ever supposed to run a single time, to initialize the editor array to the target (from then on out, the target's array is sourced from the editor array).

If we initialize `editorTriggeredObjects` to `null` then do a null check here, we'll effectively do the same thing, but ensure we only do it a single time. Better yet, just toss this in `public void Awake()`, which is run once after the target is set, and do the one-time initialization there.